_posts/blog/2018-09-18/2018-09-18-5 -tips-to-make-your-openbazaar-listings-pop.markdown

```diff
@@ -0,0 +1,158 @@
+---
+title: "<span class='post-title-extra'>5 Tips</span> To Make Your OpenBazaar Listings <span class='post-title-extra'>Pop</span>"
+layout: post
+date: '2018-09-18 06:30:00 -0300'
+social_title: '5 Tips To Make Your OpenBazaar Listings POP!'
+social_description: 'With OpenBazaar, the largest decentralized marketplace in the world, you can build your store and listings faster than on any other network and we have some tips that will help your store stand out.'
+social_image: OpenBazaar Store on OpenBazaar.png
+---
+
+_By Monique Boediono, OpenBazaar UX Designer_
+
+Ecommerce is an exciting and flexible way to start your own business. With some ecommerce business models you can even earn money from the comfort of your own home. While there are many opportunities to get started though, there are also many ways to get lost in the crowd. 
+
+![OpenBazaar Store on OpenBazaar](OpenBazaar Store on OpenBazaar.png "OpenBazaar Store on OpenBazaar")
+
+With OpenBazaar, the world's largest decentralized marketplace, you can build your store and listings faster than on any other network and we have some tips that will help your store stand out.
```

Comment:
Same as in the social network, I think it would read better if this was broken into two sentences: "any other network. We have some..."